content/en/contribute/medic/product-development-process/product-trio/product-trio-activities.md

```diff
@@ -3,19 +3,19 @@ title: "Product Trio Activities"
 linkTitle: "Product Trio Activities"
 weight: 1
 description: >
-  Common Activities of Product Trio 
+  Common Activities of the Product Trio 
```

Comment:
A lot of this is out of date too :-|

Reply:
Do you think that you can suggest changes via a PR to be merged into this branch? 🙏